GovUkDesignSystemComponents/Table.cshtml

```diff
@@ -0,0 +1,88 @@
+﻿@model GovUkDesignSystem.GovUkDesignSystemComponents.TableGovUkViewModel
+@using GovUkDesignSystem.GovUkDesignSystemComponents
+@{
+    string classNames =
+        $"govuk-table "
+        + $"{(Model == null || String.IsNullOrEmpty(Model.Classes) ? string.Empty : Model.Classes)}";
+
+    string captionClassNames =
+        $"govuk-table__caption "
+        + $"{(Model == null || String.IsNullOrEmpty(Model.CaptionClasses) ? string.Empty : Model.CaptionClasses)}";
+
+    List<TableCellViewModel> headerRow = new List<TableCellViewModel>();
+
+    if (Model.FirstCellIsHeader)
```

Comment:
I don't think this is what FirstCellIsHeader is for. It looks like it applies to the first cell in all rows (so the first column is also done as a header)

Reply:
Definitely, sorry I misunderstood it. Done, thanks